bin/prefix-vendor-namespace.php

```diff
@@ -193,7 +234,7 @@ function remove_file_autoloads( string $file, array $composer_autoload, string $
 
 	$modified = false;
 	foreach ( $json['packages'] as $key => $package ) {
-		if ( 0 !== strpos( $package['name'], $package_name ) ) {
+		if ( strpos( $package['name'], $package_name ) ) {
```

Comment:
I would leave this as a strict check because `strpos` can return false which means something completely different to found at 0. It's also causing the following error when I run the script:

```
Autoloaded file "vendor/google/apiclient/src/aliases.php" should be included in composer.json
```